SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
These tests should IMO be eventually replaced by slightly higher level ones, that test the consequence of updating the Components rather than the operation itself. They are ok for now, as we are building the system, but it might be worth adding a comment, that maintaining them would have low value, and they should be eventually phased out.